content/governing-content/urgent-content-requests/prepare-urgent-request/content-production-template.md

```diff
@@ -10,7 +10,7 @@ Include fields for:
 - user need or key message
 - related links to policies and other content
 - contact details (and an abstract if needed to go with accessible downloads)
-- a [readability rating]() check. Recommend SAS Writing Reviser, a free add-on for Microsoft Word. The tool highlights passive tense as you write.
+- a [readability rating](https://guides.service.gov.au/content-guide/writing-style/#readability) check. Recommend SAS Writing Reviser, a free add-on for Microsoft Word. The tool highlights passive tense as you write.
```

Comment:
`Recommend SAS...` sounds like a comment from the author, is there a missing word?

Reply:
Could be, will check.